vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow/src/main/java/com/vaadin/flow/component/spreadsheet/Spreadsheet.java

```diff
@@ -4831,7 +4839,7 @@ private void registerCustomComponent(Component component) {
     }
 
     private void unRegisterCustomComponent(PopupButton component) {
-        getElement().removeProperty("popupbuttons");
+        registerCustomComponent(component);
```

Comment:
Calling `registerCustomComponent` for now, because `removeProperty` will actually break if, eg., only one `PopupButton` instance is removed.

Reply:
Why not change the implementation, having a property with all the popups seems a bit weird, I would call a client method for adding/removing popups

Reply:
Changed the approach and now it adds/remove popup button through calls to each button instead.